test/FastTests/Voron/Trees/TreeRenaming.cs

```diff
@@ -53,7 +51,7 @@ public void CanRenameTree()
             }
         }
 
-        [Fact]
+        [RavenFact(RavenTestCategory.Voron)]
```

Comment:
Since you switched to `RavenFact` please adjust - [AllTestsShouldUseRavenFactOrRavenTheoryAttributes](https://jenkins.hibernatingrhinos.com/job/PR_Tests_Debug/133/testReport/junit/SlowTests.Tests/TestsInheritanceTests/AllTestsShouldUseRavenFactOrRavenTheoryAttributes/) test